library/src/scala/quoted/Quotes.scala

```diff
@@ -53,7 +53,16 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
      *  Emits an error and throws if the expression does not represent a value or possibly contains side effects.
      *  Otherwise returns the value.
      */
-    def valueOrError(using FromExpr[T]): T =
+    @deprecated("Use valueOrThrow", ">3.0.0-RC2")
+    inline def valueOrError(using inline e: FromExpr[T]): T =
```

Comment:
```suggestion
    def valueOrError(using FromExpr[T]): T =
```

Reply:
If `3.0.0-RC2` gets bumped to `3.0.0` then this would be a binary incompatible change. A macro library compiled with an older compiler (i.e. `3.0.0`) might actually call `valueOrError` in the bytecode.